### ISSUE TITLE
[NBug] Error reading the Z:\Git\te_V_CAMPARI\ directory.

### ISSUE THREAD

**Issue by alonlavian** [gitextensions/gitextensions#6676]
## Error Details
```
System.IO.FileNotFoundException: Error reading the Z:\Git\te_V_CAMPARI\ directory.
   at System.IO.FileSystemWatcher.StartRaisingEvents()
   at GitUI.CommandsDialogs.BrowseDialog.GitStatusMonitor.Update()
   at System.Windows.Forms.Timer.OnTick(EventArgs e)
   at System.Windows.Forms.Timer.TimerNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Environment
- Git Extensions 3.1.0.5877
- Build bb1687f6bd0450d0c1a2bae10a37c65493fc1294
- Git 2.18.0.windows.1
- Microsoft Windows NT 10.0.15063.0
- .NET Framework 4.7.3163.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Please fill in all required information and ping us back to re-open.

**Comment by gerhardol** [comment]
Maybe related to https://github.com/gitextensions/gitextensions/issues/6549#issuecomment-499380308
Potentially fixed in #6560

**Comment by gerhardol** [comment]
Duplicate of #6812
